Snippets/Snippets_6/Transports/SqlServer_3/MultiSchema.cs

```diff
@@ -0,0 +1,42 @@
+﻿namespace Snippets5.Transports.SqlServer_2
+{
+    using NServiceBus;
+    using NServiceBus.Transports.SQLServer;
+
+    class MultiSchema
+    {
+        void NonStandardSchema()
+        {
+            EndpointConfiguration endpointConfiguration = new EndpointConfiguration();
+
+            #region sqlserver-non-standard-schema 3
+
+            endpointConfiguration.UseTransport<SqlServerTransport>()
+                .DefaultSchema("myschema");
+
+            #endregion
+        }
+
+        void OtherEndpointConnectionParamsPull()
+        {
+            EndpointConfiguration endpointConfiguration = new EndpointConfiguration();
+            #region sqlserver-multischema-config-pull
+
+            endpointConfiguration.UseTransport<SqlServerTransport>()
+                .UseSpecificSchema(tn =>
+                {
+                    if (tn == "sales")
```

Comment:
I think it would be nice to show that tn here is not only endpoint name but also `error` and we can define here a custom schema for `error` as well.